<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-233-204" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" tests="6" failures="0" timestamp="2022-09-23T12:23:23 UTC" time="60.347" errors="0">
  <testcase name="invokeAsyncTestWith_429" time="5.054" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.077" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_503" time="40.055" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_502" time="13.042" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.046" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504" time="2.073" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
